utils/users.js

```diff
@@ -140,6 +140,14 @@ function getUsernamesFromPRs(allPRs) {
   return usernames;
 }
 
+const checkRoleToUpdate = async (userData, newRoles) => {
```

Comment:
Unclear what is the purpose of this function. Could you add a jsdoc comment please?

Reply:
Okk I will add  jsdoc comment

Reply:
Done with changes.